dff/context_storages/database.py

```diff
@@ -25,32 +27,53 @@ class DBAbstractContextStorage(ABC):
     def __init__(self) -> None:
         pass
 
+    def __getitem__(self, key: Any) -> Context:
+        return asyncio.run(self.getitem(key))
+
     @abstractmethod
-    def __getitem__(self, key: str) -> Any:
+    async def getitem(self, key: Any) -> Context:
         raise NotImplementedError
 
+    def __setitem__(self, key: Any, value: Context):
+        return asyncio.run(self.setitem(key, value))
+
     @abstractmethod
-    def __setitem__(self, key: str, value: dict) -> None:
+    async def setitem(self, key: Any, value: Context):
         raise NotImplementedError
 
+    def __delitem__(self, key: str):
+        return asyncio.run(self.delitem(key))
+
     @abstractmethod
-    def __delitem__(self, key: str) -> None:
+    async def delitem(self, key: str):
         raise NotImplementedError
 
-    @abstractmethod
     def __contains__(self, key: str) -> bool:
-        raise NotImplementedError
+        return asyncio.run(self.contains(key))
 
     @abstractmethod
+    async def contains(self, key: str) -> bool:
+        raise NotImplementedError
+
     def __len__(self) -> int:
+        return asyncio.run(self.len())
+
+    @abstractmethod
+    async def len(self) -> int:
```

Comment:
```suggestion
    async def async_contains(self, key: str) -> bool:
        raise NotImplementedError

    def __len__(self) -> int:
        return asyncio.run(self.len())

    @abstractmethod
    async def async_len(self) -> int:
```
Should  we add prefix for all `async_` methods? Like above